servant-client/test/Servant/ClientSpec.hs

```diff
@@ -90,7 +90,7 @@ type TestHeaders = '[Header "X-Example1" Int, Header "X-Example2" String]
 
 type Api =
        "get" :> Get '[JSON] Person
-  :<|> "deleteEmpty" :> Delete '[] ()
+  :<|> "deleteEmpty" :> Delete '[JSON] ()
```

Comment:
Well, not sure about the `'[]` - we still need explicit content-types even for `NoContent`. But this brought up a couple of other issues - give me a few min.